libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -56,7 +56,7 @@ const AP_Param::GroupInfo AP_TECS::var_info[] = {
     // @Range: 0.0 0.5
     // @Increment: 0.02
     // @User: Advanced
-    AP_GROUPINFO("INTEG_GAIN", 4, AP_TECS, _integGain, 0.1f),
+    AP_GROUPINFO("INTEG_GAIN", 4, AP_TECS, _integGain, 0.3f),
```

Comment:
do you think we need to do any param conversion from existing params? Note that hardly anybody tunes TECS ....

Reply:
Probably not. The new TECs can use a larger I value without running into limit cycle issues, hence the change to the default value. If the user is already using a larger I value, then we don't need to change their parameter value.